tests/YearWeekTest.php

```diff
@@ -395,4 +397,30 @@ public function testToString(int $year, int $week, string $expected)
         $yearWeek = YearWeek::of($year, $week);
         $this->assertSame($expected, (string) $yearWeek);
     }
+
+    public function testNow()
+    {
+        $timeZone = TimeZone::parse('Asia/Taipei');
+        $yearWeek = YearWeek::now($timeZone);
```

Comment:
You should provide a Clock here, and check that the resulting YearWeek is in line with the clock!

Reply:
Can you give some example about this?

Thanks.

Reply:
You can set set timestamp that `now()` will use, this way:

```php
$clock = new FixedClock(Instant::of(2000000000));
$yearWeek = YearWeek::now($timeZone, $clock);
```